src/monitors/LoopMonitor.v3

```diff
@@ -183,3 +188,21 @@ def report(entries: Vector<LmEntry>) {
 	}
 
 }
+def reportCsv(entries: Vector<LmEntry>) {
+	OUT.puts("func,pc,end_pc,count\n");
+	for (i < entries.length) {
+		var e = entries[i];
+		if (e.count == 0 && !showZeroCounts) continue;
+		//add each line of data so the out buffer
+		OUT.put1("%q", e.func.render(e.module.names, _))
+			.putc(',')
+			.putd(e.pc)
```

Comment:
Whether there is a `+` before `pc` should be consistent between all the monitors; I vote to not have it, since it's a numerical quantity meant to be consumed by machines.

Reply:
I'll change the ones I've interacted with on the csv ouputs